package-lock.json

```diff
@@ -1,8 +1,6177 @@
 {
```

Comment:
It's part of the repo upstream. I didn't change that here. Just left it to avoid increasing delta from the original project so we can sync changes easier.